react/features/recording/components/Recording/AbstractStartRecordingDialog.js

```diff
@@ -132,6 +138,7 @@ class AbstractStartRecordingDialog extends Component<Props, State> {
      */
     constructor(props: Props) {
         super(props);
+        const supportsLocalRecording = browser.isChromiumBased() && !browser.isElectron() && !isMobileBrowser();
```

Comment:
Don't we have this check elsewhere? Looks like it would be useful to move to a function perchance?